packages/app-store/slack/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "/*": "Don't modify slug - If required, do it using cli edit command",
+  "name": "Slack",
+  "slug": "slack",
+  "type": "slack_messaging",
+  "logo": "icon.svg",
+  "url": "https://example.com/link",
```

Comment:
this should be a Cal.com link

Reply:
Should be link to contributor's profile as I suggested above.